tests/CMakeLists.txt

```diff
@@ -37,3 +37,8 @@ add_integration_test_2("examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py")
 add_integration_test("examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py")
 add_integration_test("examples/FCCee/flavour/Bc2TauNu/analysis_B2TauNu_truth.py")
 add_integration_test("examples/FCCee/test/jet_constituents.py")
+find_package(ONNXRuntime)
```

Comment:
Not sure if this is the most elegant way of checking whether FCCAnalyses library was linked against ONNXRuntime...

Reply:
Yes,  better to add an `option` and then just put this inside `if(WITH_ONNX)`, see my comment above